core/Repository/Utils/TaskRepositoryUtils.h

```diff
@@ -0,0 +1,21 @@
+//
+// Created by valeriisudakov on 19.10.20.
+//
+
+#ifndef TASKMANAGER_CORE_REPOSITORY_UTILS_TASKREPOSITORYUTILS_H_
+#define TASKMANAGER_CORE_REPOSITORY_UTILS_TASKREPOSITORYUTILS_H_
+#include "Repository/DTO/TaskRepositoryDTO.h"
+#include "Repository/Task/TaskEntity.h"
+#include "Repository/Task/ID/TaskIDGenerate.h"
+
+namespace TaskRepositoryUtils {
```

Comment:
Rude violation of namespace naming rules.
https://google.github.io/styleguide/cppguide.html#Namespace_Names

Reply:
fixed.